IntegrationTests/UI.IntegrationTests/UITest.cs

```diff
@@ -77,6 +77,29 @@ void HandleApplicationIdle(object sender, EventArgs e)
             }
         }
 
+        public static void RunControl<T>(
+            Func<Form, T> createControl,
+            Func<T, Task> runTestAsync)
+            where T : Control
+        {
+            T control = null;
```

Comment:
[nit]
```suggestion
            T control = default;
```

Reply:
My classical view: I don't want a potentially defined default object. `null` emphasizes that the below `Dispose` must be called with the conditional operator.
Are you going to turn `Control` into a non-nullable reference type?

Reply:
`null` is for reference types. `default` is for value types and unconstrained generics.